### PR TITLE
Added : Badge component

### PR DESCRIPTION
**Pull request by ayush65**

### CODE REVIEW

File: components/Avatar/Avatar.css [hunk 1]
```diff
@@ -18,8 +18,8 @@
     border-bottom-right-radius: 10px;
     width: 15.625rem;
     grid-area: aside;
-    height: 1800px;
```
Comment (HarshithaKonduru214) [inline components/Avatar/Avatar.css:21]: Try to use consistent units.

**Review by HarshithaKonduru214 — APPROVED** [review]

File: components/Avatar/Avatar.css (hunk 1 above)
Comment (HarshithaKonduru214) [reply components/Avatar/Avatar.css:21]: Rest everything looks good to me.

Comment (ayush65) [reply components/Avatar/Avatar.css:21]: yea i have updated that thanks for reviewing

**Review by shraddha15-creator — APPROVED** [review]

**Review by NtshVrm — APPROVED** [review]

**Review by mimansha-swarup — APPROVED** [review]

**Review by narharikale — APPROVED** [review]
looks great👍

**Comment by Himanshu-Dhawale** [comment]
Looks good

**Pull request merged**